### DISCUSSION TITLE
Combining scanning multiple payloads and multiple endpoints

### DISCUSSION THREAD

**Discussion by TimMaasGeesteranus** [projectdiscovery/nuclei#5738] [Q&A]
### Let's say I want to scan for SLQ injections on **mywebsite.com**. 

I have a list of 2 endpoints in endpoints.txt:
- /form?name=ben&age=35&length=180 
- /login?usr=username&pass=password. 

I have a list of  3 sql injection payloads in payloads.txt: 
- sql1
- sql2
- sql3.

### Is it possible make a template such that nuclei will automatically scan 6 times? (2 endpoints * 3 payloads = 6 scans). The requests should look like this:

- mywebsite.com/form?name=sql1&age=sql1&length=sql1
- mywebsite.com/form?name=sql2&age=sql2&length=sql2
- mywebsite.com/form?name=sql3&age=sql3&length=sql3
- mywebsite.com/login?usr=sql1&pass=sql1
- mywebsite.com/login?usr=sql2&pass=sql2
- mywebsite.com/login?usr=sql3&pass=sql3

I have not been able to create this using the Nuclei payload and variable options, but maybe I'm overlooking something. The results should be scalable, so the amount of endpoints, parameters and payloads shouldn't matter

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
